Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -367,6 +367,24 @@ public extension Node where Context: HTML.BodyContext {
     }
 }
 
+// MARK: - Subresource Integrity
+
+public extension Attribute where Context: HTMLIntegrityContext {
+    /// Assign a source to the element, using its `integrity` attribute.
+    /// - parameter hash: base64-encoded cryptographic hash
+    static func integrity(_ hash: String) -> Attribute {
+        Attribute(name: "integrity", value: hash)
+    }
+}
+
+public extension Node where Context: HTMLIntegrityContext {
+    /// Assign a source to the element, using its `src` attribute.
```

Comment:
Same comment here as above, plus the `src` attribute should be changed to `integrity`.